recipes/poco/all/conanfile.py

```diff
@@ -201,11 +203,13 @@ def package_info(self):
         if self.settings.os == "Linux":
             self.cpp_info.system_libs.extend(["pthread", "dl", "rt"])
 
-        if self.settings.compiler == "Visual Studio":
+        if self.settings.os == "Windows":
```

Comment:
The `POCO_NO_AUTOMATIC_LIBS` is only valid for `Visual Studio`:
https://github.com/pocoproject/poco/blob/5e7d652c7518b3f3e9242f9618ce03ffd309a07a/NetSSL_OpenSSL/include/Poco/Net/NetSSL.h#L56-L62